core/src/main/java/ar/com/wolox/wolmo/core/util/SharedPreferencesManager.kt

```diff
@@ -0,0 +1,39 @@
+package ar.com.wolox.wolmo.core.util
+
+import android.content.SharedPreferences
+import ar.com.wolox.wolmo.core.di.scopes.ApplicationScope
+import javax.inject.Inject
+
+/**
+ * Utility class to query and store values in {@link SharedPreferences}.
+ */
+@ApplicationScope
+class SharedPreferencesManager @Inject constructor(private val sharedPreferences: SharedPreferences) {
```

Comment:
We should use the `set operator`[https://kotlinlang.org/docs/reference/operator-overloading.html#indexed] instead of custom `store` functions.

We should also use the `kotlin android extensions` that provide `SharedPreferences` utilities here.